### PR TITLE
fix(Group): Group is now handled separately

### PR DESCRIPTION
**Pull request by szemyd**
Closes #

### CODE REVIEW

File: src/krisi/evaluate/metric.py [hunk 1]
```diff
@@ -320,20 +318,6 @@ def __safe_set(
         else:
             self.__dict__[key] = result
 
-    # def __add__(self, other):
```
Comment (almostintuitive) [inline src/krisi/evaluate/metric.py:323]: is this needed?:)

**Review by almostintuitive — APPROVED** [review]
👍

File: src/krisi/evaluate/metric.py (hunk 1 above)
Comment (szemyd) [reply src/krisi/evaluate/metric.py:323]: No, that is why I deleted it! Do you need it, I can put it back.

Comment (almostintuitive) [reply src/krisi/evaluate/metric.py:323]: ohh sorry!

**Pull request merged**